estimagic/tests/optimization/test_all_algorithms_with_sum_of_squares.py

```diff
@@ -647,6 +646,74 @@ def test_with_sdcorr_constraint_bounds_distance(
     )
 
 
+@pytest.mark.slow
+@pytest.mark.parametrize(
+    "algo, direction, crit, deriv, crit_and_deriv", bound_cases
+)  # ================================
+def test_with_decreasing_constraint_and_fixes(
+    algo, direction, crit, deriv, crit_and_deriv
+):
+    params = pd.DataFrame(data=[[1], [4], [4], [2], [1]], columns=["value"])
+
+    constraints = [
+        {"loc": [1, 2, 3, 4], "type": "decreasing"},
+        {"loc": 2, "type": "fixed", "value": 4},
+    ]
+
+    optimize_func = minimize if direction == "minimize" else maximize
+
+    res = optimize_func(
```

Comment:
Given that the expected result is hard coded to the argmin the test should fail in the case of maximize. Maybe simply `pass` in the case of maximize?

Reply:
No, we switch the sign of the criterion function in that case. Same is done in all other test cases in that module.